Lib/idlelib/idle_test/test_textview.py

```diff
@@ -96,5 +96,52 @@ def test_view_file(self):
         self.assertIsNone(view)
 
 
+class ButtonClickTextViewTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
I don't think this and tearDownClass are needed.  The module root should work.  Just pass '_utest=True' in both calls.